### ISSUE TITLE
CXX from pythranrc

### ISSUE THREAD

**Issue by diorcety** [serge-sans-paille/pythran#956]
When compiling directly using pythran command line doesn't use CXX and CC from pythranrc file

**Comment by serge-sans-paille** [comment]
Thanks @ashwinvis for the feedback. Looks like you're not using PythranBuildExt in  your setup.py, so the CXX override behavior is not activated. Funny enough, if I recall correctly, you were the one to advocate for that feature :-/

Am I missing something?

**Comment by serge-sans-paille** [comment]
Adding ``cmdclass={"build_ext": PythranBuildExt},`` to your setup.py instead fixes the issue.

**Comment by ashwinvis** [comment]
I am certainly missing something... Need some more time to track it down. My reply above was when I used PythranBuildExt.

I guess the catch i Pythran extensions arr built after C++ Cython extensions in fluidfft.

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by serge-sans-paille** [comment]
And anyway, we should not *ignore* CXX if ``PythranBuildExt`` is not used, I'll check that too.

**Comment by ashwinvis** [comment]
No you are right.... It was my mistake. I was subclassing PythranBuildExt like:

```python
        from pythran.dist import PythranBuildExt

        class fluidfft_build_ext(build_ext, PythranBuildExt):
               pass
```

Certainly didn't work as intended. Worked when I tried `PythranBuildExt` explicitly.

**Comment by serge-sans-paille** [comment]
And CXX is not ignored, I just forgot to set CC too, merging in that case

**Comment by serge-sans-paille** [comment]
Fixed by #958
